types/rlp_encoding_test.go

```diff
@@ -162,6 +162,67 @@ func TestRLPMarshall_And_Unmarshall_TypedTransaction(t *testing.T) {
 
 		unmarshalledTx.ComputeHash()
 		assert.Equal(t, originalTx.Type, unmarshalledTx.Type)
+		assert.Equal(t, originalTx.Hash, unmarshalledTx.Hash)
+	}
+}
+
+func TestRLPMarshall_Unmarshall_Missing_Data(t *testing.T) {
+	txTypes := []TxType{
```

Comment:
Consider adding parallel execution?

```suggestion
func TestRLPMarshall_Unmarshall_Missing_Data(t *testing.T) {
        t.Parallel()
	txTypes := []TxType{
```

Reply:
fixed 673cce8